dictionaries/software-terms/src/software-terms.txt

```diff
@@ -2192,3 +2192,5 @@ zip
 zlib
 ZPool
 ZPools
+sysmon
```

Comment:
`sysmon` is more of a tool than a software concept. Would you please add it to [software-tools.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-tools.txt)?